File: Python/Bubble_sort.py
def bubbleSort(arr):
    n = len(arr)
    for i in range(n):
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
               
    return arr
n=int(input("enter length"))
arr=[]
for i in range(n):
    arr.append(int(input("enter the element")))
print("before sort",arr)
print("after sort",bubbleSort(arr))

File: Java/bubblesort.java
import java.util.Scanner;

class bubblesort {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Input the number of elements in the array");
        int length = sc.nextInt();

        int[] array = new int[length];

        // Taking the user input
        for (int i = 0; i < length; i++) {
            System.out.println("Input the next element (integer) of the array.");
            array[i] = sc.nextInt();
        }

        // Sorting the array
        for (int i = 0; i < length; i++) {
            for (int j = i + 1; j < length; j++) {
                // Ascending order
                // repalace > to < for descending order
                if (array[i] > array[j]) {
                    int temp = array[i];
                    array[i] = array[j];
                    array[j] = temp;
                }
            }
        }

        // Priniting the sorted array
        System.out.println("Sorted Array : ");
        for (int i = 0; i < length; i++)
            System.out.println(array[i]);
    }
}